From aecbde8771d02ea17ed83a2ccdf1051adae7e656 Mon Sep 17 00:00:00 2001
From: Ashutosh Chauhan <hashutosh@apache.org>
Date: Tue, 21 Apr 2015 17:07:29 +0000
Subject: [PATCH 0593/1164] CDH-40141: more HIVE-10396 : decimal_precision2.q
 test is failing on trunk (Jason Dere via Ashutosh
 Chauhan)

Change-Id: I12e0dae6877e7e33233432af94a0e66c4cd4c8e6
git-svn-id: https://svn.apache.org/repos/asf/hive/trunk@1675157 13f79535-47bb-0310-9956-ffa450edef68
---
 .../clientpositive/decimal_precision2.q.out        |    8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/ql/src/test/results/clientpositive/decimal_precision2.q.out b/ql/src/test/results/clientpositive/decimal_precision2.q.out
index 3b7d489..7321889 100644
--- a/ql/src/test/results/clientpositive/decimal_precision2.q.out
+++ b/ql/src/test/results/clientpositive/decimal_precision2.q.out
@@ -37,7 +37,7 @@ STAGE PLANS:
           Row Limit Per Split: 1
           Statistics: Num rows: 1 Data size: 1 Basic stats: COMPLETE Column stats: COMPLETE
           Select Operator
-            expressions: 100.000 (type: decimal(3,0))
+            expressions: 100 (type: decimal(3,0))
             outputColumnNames: _col0
             Statistics: Num rows: 1 Data size: 112 Basic stats: COMPLETE Column stats: COMPLETE
             ListSink
@@ -59,7 +59,7 @@ STAGE PLANS:
           Row Limit Per Split: 1
           Statistics: Num rows: 1 Data size: 1 Basic stats: COMPLETE Column stats: COMPLETE
           Select Operator
-            expressions: 0.000 (type: decimal(1,0))
+            expressions: 0 (type: decimal(1,0))
             outputColumnNames: _col0
             Statistics: Num rows: 1 Data size: 112 Basic stats: COMPLETE Column stats: COMPLETE
             ListSink
@@ -81,7 +81,7 @@ STAGE PLANS:
           Row Limit Per Split: 1
           Statistics: Num rows: 1 Data size: 1 Basic stats: COMPLETE Column stats: COMPLETE
           Select Operator
-            expressions: 0.100 (type: decimal(1,1))
+            expressions: 0.1 (type: decimal(1,1))
             outputColumnNames: _col0
             Statistics: Num rows: 1 Data size: 112 Basic stats: COMPLETE Column stats: COMPLETE
             ListSink
@@ -103,7 +103,7 @@ STAGE PLANS:
           Row Limit Per Split: 1
           Statistics: Num rows: 1 Data size: 1 Basic stats: COMPLETE Column stats: COMPLETE
           Select Operator
-            expressions: 0.010 (type: decimal(2,2))
+            expressions: 0.01 (type: decimal(2,2))
             outputColumnNames: _col0
             Statistics: Num rows: 1 Data size: 112 Basic stats: COMPLETE Column stats: COMPLETE
             ListSink
-- 
1.7.9.5

